src/action.ts

```diff
@@ -128,7 +130,13 @@ export default async function main() {
       ? `pre${bump || defaultBump}`
       : bump || defaultBump;
 
-    const incrementedVersion = inc(previousVersion, releaseType, identifier);
+    let incrementedVersion: string | null;
+
+    if (isPullRequest) {
+      incrementedVersion = inc(previousVersion, releaseType);
+    } else {
+      incrementedVersion = inc(previousVersion, releaseType, identifier);
+    }
```

Comment:
I don't really understand this part: if the releaseType isn't `preX`, then the identifier shouldn't matter?

Reply:
I'll test without this block, one of the changes I made after this may negate the need for this.

Reply:
@mathieudutour I've tested and confirmed it works without this change, I've reverted this change.